CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,47 @@
+# Code de conduite de StrasbourgJS
+
+Tous les participants, conférenciers, partenaires et bénévoles de notre conférence sont tenus d'accepter le code de conduite suivant. Les organisateurs feront respecter ce code durant les événements et sur les réseaux sociaux. Nous attendons de tous les participants qu'ils coopèrent afin de garantir un environnement sûr pour tous.
+
+## Mission
+
+Le groupe StrasbourgJS s'engage à créer une communauté saine, respectueuse, sans distinction de sexe, d'identité, d'expression sexuelle, d'orientation sexuelle, de handicap, d'apparence physique, d'âge, de taille corporelle, de race, de nationalité ou de croyances religieuses.
+
+## Nous ne tolérons pas...
+
+Le harcèlement comprend les commentaires offensants, verbaux ou écrits, liés au sexe, à l'identité ou à l'expression sexuelle, à l'orientation sexuelle, au handicap, à l'apparence physique, à l'âge, à la taille, à la race, à la nationalité ou aux croyances religieuses, l'intimidation délibérée, les menaces, le harcèlement, le suivi, la photographie ou l'enregistrement harassant, le contact physique inapproprié et l'attention sexuelle non souhaitée. Le langage et les images à caractère sexuel ne sont pas appropriés.
+
+Ces règles s'appliquent lors des conférences, les ateliers, les événements, Twitter, Meetup.com, GitHub et autres médias en ligne.
```

Comment:
[...] les évènements ainsi qu'à l'ensemble des médias rattachés au groupe StrasbourgJS permettant toute forme d'échange publics ou privé et les plateformes de promotion de nos activités.

Comme ça on est future-proof.

Reply:
@KeziahMoselle publics sans "s" !

Reply:
Thank you @edimitchel